arcgis-ios-sdk-samples/Scenes/Extrude graphics/ExtrudeGraphicsViewController.swift

```diff
@@ -99,7 +99,7 @@ class ExtrudeGraphicsViewController: UIViewController {
     //add a graphic to the graphics overlay for the given polygon
     private func addGraphicForPolygon(_ polygon: AGSPolygon) {
         
-        let rand = arc4random_uniform(UInt32(self.maxHeight))
+        let rand = Int.random(in: 0...maxHeight)
```

Comment:
It might be worth pointing out that `maxHeight` is now a possible random number whereas before it wasn't. I don't see a problem with that though.

Reply:
Nice catch. I think it's an extremely minor behavior change so I'm not worried about it.